Create v0.0.1-intro.md (#1)
roast-cms/bloat: v0.0.1-intro.md

This is the first entry into a new blog about building things on the web. Yeah, there are tons of blogs like that out there — but that isn't a deterrent for me today. The Internet does not have _everything_.

> I decided to name this blog "**bloat**." It's a shorter way to utter and spell "over-engineering," which often plagues my personal projects. I will talk about that here a lot. I've also decided to leave the name of the blog _lowercase_, so it looks more trendy and nerdy — like a variable declaration or something.

Author: ```suggestion
I decided to name this blog "**bloat**." It's a shorter way to utter and spell "over-engineering," which often is the theme of my personal projects. I will talk about that here a lot.
```

Being a kid, I blamed our stressful financial state on their chosen vocation. Which steered me more towards science and computers; I wasn't very good, just slightly more interested in technology than the average kid in the early '00s.

In college, my habits and relationships got me closer to art again. I formed a few bands, and started to design stuff on a computer, including sound for our singles and [animation](https://vimeo.com/13145639) for our videos. About 15 years later, my drive to make artsy things lead me to build [Analog.Cafe](https://www.analog.cafe/about), a blog about film photography. That blog is built on the "in-house" (I'll explain later what I mean by that) CMS that I hope to use for this blog as well.

Professionally, I've been working as a web developer since graduating the University of Toronto. There were a few small startups, including businesses I started solo and with friends. Some jobs allowed me to travel extensively. You may even recognize some brands I work(ed) for: Rakuten and WebMD. I went through two acquisitions, one of which paid off — nothing crazy, less than $100K (I will talk about that also).

# Next steps.

> Today, I'm a little busy. I live with my wife and two dogs in a small apartment; one of them is a puppy, so he needs a lot of walks and attention. I'm also spending a lot of time on my other blog (and all the associated projects), Analog.Cafe. And, of course, there's that full-time job that pays for all of this. So I am making no public commitments about voluminous content here. But I hope to share something valuable whenever I can — maybe once a month? Who knows.

Author: ```suggestion
These days, I'm a little busy. I live with my wife and two dogs in a small apartment; one of them is a puppy, so he needs a lot of walks and attention. I'm also spending a lot of time on my other blog (and all the associated projects), Analog.Cafe. And, of course, there's that full-time job that pays for all of this. So I am making no public commitments about voluminous content here. But I hope to share something valuable whenever I can — maybe once a month? Who knows.
```
